app/src/main/java/org/schabi/newpipe/settings/SettingsActivity.java

```diff
@@ -56,11 +56,11 @@ protected void onCreate(final Bundle savedInstanceBundle) {
                 SettingsLayoutBinding.inflate(getLayoutInflater());
         setContentView(settingsLayoutBinding.getRoot());
 
-        setSupportActionBar(settingsLayoutBinding.toolbarLayout.toolbar);
+        setSupportActionBar(settingsLayoutBinding.settingsToolbarLayout.toolbar);
```

Comment:
where does this come from?

(discard my previous comment, if you got an email notification about the layout weight. I was not understanding the commit :D )

Reply:
We have multiple of the same ids (e.g. fragment_holder, toolbar_layout, etc.) throughout different layouts, which is not good practice :(

Of course this will probably never happen, but if this app were large enough to have tens of layouts with the same ids, it could become extremely difficult to understand in the code where specific layouts/ids are being used, and subsequently untangle them (take a look at [Slide](https://github.com/ccrama/Slide/tree/master/app/src/main/res/layout) for a perfect example).

Therefore, this is a start to making sure that each and every layout has its own unique set of ids :)